string file_prefix
string[] topic_list
bool stop
---